common/exceptions.py

```diff
@@ -16,30 +18,31 @@ def __str__(self):
         return message
 
     def to_json(self):
+        """Converts the exception to json format."""
         response = {"error": self.msg}
         response.update(self.kwargs)
         return response
```

Comment:
Not really a part of this PR, but this could be a one-liner:
```python
    return {"error": self.msg, **kwargs}
```

Reply:
Will fix this in #216